cmd/svcat/binding/bind_cmd.go

```diff
@@ -66,7 +66,7 @@ func NewBindCmd(cxt *command.Context) *cobra.Command {
 		PreRunE: command.PreRunE(bindCmd),
 		RunE:    command.RunE(bindCmd),
 	}
-	command.AddNamespaceFlags(cmd.Flags(), false)
+	bindCmd.AddNamespaceFlags(cmd.Flags(), false)
```

Comment:
all of the renames here that move from global command package to a local var. This all ends up doing the same thing?

Reply:
The renames are for consistency. The real refactor is this commit:

https://github.com/kubernetes-incubator/service-catalog/pull/2115/commits/74360df6ff16cfe9cecaee137d45c3cca8a0b908#diff-850d429a6c9ee1abf27495e774833607